src/index.js

```diff
@@ -148,7 +166,7 @@ lockfile.wait()
     } else if (action === ACTIONS.COMPLETE) {
       return bootstrap.complete();
     } else {
-      return bootstrap.existing();
+      return bootstrap.existing(mode.upgradeDocuments[0]);
```

Comment:
My hope here was that LEGACY_0_8_UPGRADE_DOC codepath is dead and I can remove it. If that's the case, then upgradeDocument can be a single file instead of an array and this problem goes away.